addons/hw_drivers/tools/helpers.py

```diff
@@ -702,3 +703,16 @@ def reset_log_level():
             'log_handler': ':WARNING',
             'log_level': 'warn',
         })
+
+
+def get_raspberry_pi_model():
```

Comment:
```suggestion
@cache
def get_raspberry_pi_model():
```

maybe we could cache this if we need to reuse it one day?

Reply:
or make it a constant rather than a function ?